src/components/Graph/Graph.js

```diff
@@ -0,0 +1,264 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import React, { useState } from 'react';
+import { View, PanResponder } from 'react-native';
+import Svg, {
+  Polyline,
+  Line,
+  Circle,
+  Path,
+  ForeignObject,
+} from 'react-native-svg';
+import { withTheme } from 'styled-components/native';
+import { BaseText } from 'components/Typography';
+import { getThemeColors } from 'utils/themes';
+import type { Theme } from 'models/Theme';
+
+
+type DataPoint = {
+  x: number,
+  y: number,
+};
+
+type DataPoints = DataPoint[];
+
+type Props = {
+  data: DataPoints,
```

Comment:
Maybe mention the [0, 1] requirement for data points (and if there are any others).